src/components/Nav.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { Link } from 'react-router-dom';
+
+export interface NavState {
+    page: string,
+}
+
+export default class Nav extends React.Component<{}, NavState> {
+    constructor(props: {}) {
+        super(props)
+        this.state = {
+            page: 'home',
+        }
+    }
+
+    render() {
+        return(
+            <nav>
+                <Link className = 'logo' to='/' onClick={() => this.setState({page: 'home'})}>
+                <h3>Logo</h3>
+                </Link>
+                {/* There is probably a better way to make a nav item appear "active"
+                It would probably be better to use the stylesheet for that somehow
+                Please let me know if you know a better way
+                */}
+                <ul className='nav-links'>
+                    <Link className = 'link' to='/' 
```

Comment:
navlink allows an active class to be set automatically. we dont need all this state stuff

https://reactrouter.com/web/api/NavLink